src/react/deckgl.js

```diff
@@ -126,6 +128,18 @@ export default class DeckGL extends React.Component {
       );
     }
 
+    // TODO: add handlers on demand at runtime, not all at once on init
+    this.eventManager = createEventManager(canvas, {})
+    .on({
+      click: this._onClick,
```

Comment:
@shaojingli and I discussed. He'll add detail to the RFC, but to recap: here's a suggestion for rethinking how application developers may handle user input:

1. Selection-driven: App dev adds selection handlers that return elements selected, deselected, or soft-selected (e.g. hovered). This is a more declarative style of handling user input.

2. Traditional event handling: App dev adds input event handlers that return screen or model-space coordinates, and then uses those coords to query features. This is a more imperative style of handling user input.

If we add support for selection events as outlined in 1. above, we should probably still maintain the traditional event handling paradigm for backwards compatibility and because it's a more familiar paradigm. But we might consider shifting focus toward e.g. `onItemsPicked` and away from `onClick`.